backend/tests/unit/matcher/test_parsers.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import pytest
+from bs4 import BeautifulSoup
+from inspire_schemas.api import load_schema, validate
+
+from inspirehep.matcher.parsers import GrobidParser
+
+
+def test_grobid_to_reference():
+    xml = """
+<biblStruct >
+    <analytic>
+        <title/>
+        <author>
+            <persName xmlns="http://www.tei-c.org/ns/1.0"><surname>Graff</surname></persName>
+        </author>
+    </analytic>
+    <idno type="DOI">10.1007/JHEP11(2010)093</idno>
+    <idno type="arxiv">1009.1212</idno>
+    <idno type="report">ESS 2013-001</idno>
```

Comment:
* are you sure you get `type="report"`? in the example I looked at it was just a bare `<idno>` without `type` attribute.
* I think you should use more realistic examples, the `idno` tags don't seem to be in the right place (they should be under `analytic` according to examples I looked at).

Reply:
The examples are based on GROBID responses, some of them indeed I have enchached with more metadata to edge more cases but the structure is exaclty as GROBID responses. 

Could you please provide these examples with `analytic`? Please see below some of the queries that I used:

1. 
```bash
curl 'https://grobid.inspirebeta.net/api/processCitation' --data-raw 'citations=[28] K. Ito, H. Nakajima, T. Saka, and S. Sasaki, “N=2 Instanton Effective Action in
Ω-background and D3/D(-1)-brane System in R-R Background,” JHEP 11 (2010) 093,
arXiv:1009.1212 [hep-th].'
```
```xml
<biblStruct >
        <monogr>
                <title level="m" type="main">N=2 Instanton Effective Action in Ω-background and D3/D(-1)-brane System in R-R Background</title>
                <author>
                        <persName xmlns="http://www.tei-c.org/ns/1.0"><forename type="first">K</forename><surname>Ito</surname></persName>
                </author>
                <author>
                        <persName xmlns="http://www.tei-c.org/ns/1.0"><forename type="first">H</forename><surname>Nakajima</surname></persName>
                </author>
                <author>
                        <persName xmlns="http://www.tei-c.org/ns/1.0"><forename type="first">T</forename><surname>Saka</surname></persName>
                </author>
                <author>
                        <persName xmlns="http://www.tei-c.org/ns/1.0"><forename type="first">S</forename><surname>Sasaki</surname></persName>
                </author>
                <idno type="arXiv">arXiv:1009.1212</idno>
                <imprint>
                        <date type="published" when="2010" />
                        <biblScope unit="page">93</biblScope>
                </imprint>
        </monogr>
        <note>hep-th</note>
</biblStruct>
```

2. 

```bash
curl 'https://grobid.inspirebeta.net/api/processCitation' --data-raw 'citations=Phys. Lett. B 704 (2011) 223' 
```

```xml
        <analytic>
                <title/>
        </analytic>
        <monogr>
                <title level="j">Phys. Lett. B</title>
                <imprint>
                        <biblScope unit="volume">704</biblScope>
                        <biblScope unit="page">223</biblScope>
                        <date type="published" when="2011" />
                </imprint>
        </monogr>
</biblStruct>
```